examples/flower-client-authentication/client.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict
+import flwr as fl
+from flwr.common import NDArrays, Scalar
+
+from task import (
+    Net,
+    DEVICE,
+    load_data,
+    get_parameters,
+    set_parameters,
+    train,
+    test,
+)
+
+
+# Load model and data (simple CNN, CIFAR-10)
+net = Net().to(DEVICE)
+trainloader, testloader = load_data()
+
+
+# Define Flower client
+class FlowerClient(fl.client.NumPyClient):
+    def get_parameters(self, config: Dict[str, Scalar]) -> NDArrays:
+        return get_parameters(net)
+
+    def fit(self, parameters, config):
+        set_parameters(net, parameters)
+        results = train(net, trainloader, testloader, epochs=1, device=DEVICE)
+        return get_parameters(net), len(trainloader.dataset), results
+
+    def evaluate(self, parameters, config):
+        set_parameters(net, parameters)
+        loss, accuracy = test(net, testloader)
+        return loss, len(testloader.dataset), {"accuracy": accuracy}
+
+
+def client_fn(cid: str):
+    return FlowerClient().to_client()
+
+
+# Run via `flower-client-app client:app`
+app = fl.client.ClientApp(
```

Comment:
Let's adopt this pattern going forward (imports need to change):
```suggestion
app = ClientApp(
```